kolibri_explore_plugin/assets/src/views/CustomChannelPresentationApp.vue

```diff
@@ -131,8 +131,8 @@
 
   .custom-presentation-iframe {
     width: 100%;
-    // Remove header size (64px) and 5px for avoiding scrollbar.
-    height: calc(100vh - 64px - 5px);
+    // Remove 5px for avoiding scrollbar.
+    height: calc(100vh - 5px);
```

Comment:
For some reason this calc is not needed anymore. `height: 100vh;` works fine. The negative margin below is still needed to prevent the double scrollbar.